src/aspire/estimation/covar2d.py

```diff
@@ -37,12 +39,12 @@ def _get_mean(self, coeffs):
         """
         if coeffs.size == 0:
             raise RuntimeError('The coefficients need to be calculated first!')
-        self.as_type = coeffs.dtype
+        self.dtype = coeffs.dtype
         mask = self.basis._indices["ells"] == 0
-        mean_coeff = np.zeros((self.basis.count, 1), dtype=self.as_type)
+        mean_coeff = np.zeros((self.basis.count, 1), dtype=self.dtype)
```

Comment:
See above. Why not `dtype=coeffs.dtype`?

Reply:
As above, it is used to make consistent with other classes. Changed in the new commit # 4c22159.